modules/tftp/server.rb

```diff
@@ -7,76 +7,98 @@ class Server
     # Creates TFTP pxeconfig file
     def set mac, config
       raise "Invalid parameters received" if mac.nil? || config.nil?
-
       FileUtils.mkdir_p pxeconfig_dir
```

Comment:
I don't think this is needed anymore -- the dir is being created in write_file method.